packages/terra-signature/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "terra-signature",
+  "main": "lib/Signature.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "A signature defined by an array of line segments.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-signature",
+    "Signature",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-toolkit": "^0.x"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-base": "^0.x",
+    "terra-mixins": "^1.0.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "jquery": "^3.2.1",
+    "prop-types": "^15.5.8",
+    "terra-base": "^1.4.0",
+    "terra-mixins": "^1.0.0"
+  },
+  "scripts": {
```

Comment:
There is some clean up needed in scripts. Take a look at some of the other packages to see what's been updated. Most should be able to be copy/pasted back to here and adjusted for this package.